### PR TITLE
[zh-CN]: update translation for Broadcast Channel API - part II

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Related issues and pull requests

part of the #21059

### CODE REVIEW

File: files/zh-cn/web/api/broadcastchannel/postmessage/index.md [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
-```plain
-var str = channel.postMessage(object);
+```js-nolint
+postMessage(message)
 ```
 
+### 参数
+
+- `message`
+  - : 要发送到其他窗口的数据。数据使用[结构化克隆算法](/zh-CN/docs/Web/API/Web_Workers_API/Structured_clone_algorithm)进行序列化。这意味着您可以将各种数据对象安全地传递到目标窗口，而无需自己对其进行序列化。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcastchannel/postmessage/index.md:19]:
```suggestion
  - : 要发送到其他窗口的数据。数据使用[结构化克隆算法](/zh-CN/docs/Web/API/Web_Workers_API/Structured_clone_algorithm)进行序列化。这意味着你可以将各种数据对象安全地传递到目标窗口，而无需自己对其进行序列化。
```

File: files/zh-cn/web/api/broadcastchannel/close/index.md [hunk 2]
```diff
@@ -1,27 +1,25 @@
 ---
-title: BroadcastChannel.close()
+title: BroadcastChannel：close() 方法
 slug: Web/API/BroadcastChannel/close
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcastchannel/close/index.md:3]:
```suggestion
title: BroadcastChannel：close() 方法
slug: Web/API/BroadcastChannel/close
l10n:
  sourceCommit: 50a45d52fd9f45f1ca30b546af5920d0ccda82dc
```

File: files/zh-cn/web/api/broadcastchannel/message_event/index.md [hunk 3]
```diff
@@ -3,17 +3,17 @@ title: BroadcastChannel：message 事件
 slug: Web/API/BroadcastChannel/message_event
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcastchannel/message_event/index.md:3]:
```suggestion
slug: Web/API/BroadcastChannel/message_event
l10n:
  sourceCommit: 50a45d52fd9f45f1ca30b546af5920d0ccda82dc
```

File: files/zh-cn/web/api/broadcastchannel/messageerror_event/index.md [hunk 4]
```diff
@@ -1,42 +1,45 @@
 ---
-title: "BroadcastChannel: messageerror event"
+title: BroadcastChannel：messageerror 事件
 slug: Web/API/BroadcastChannel/messageerror_event
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcastchannel/messageerror_event/index.md:3]:
```suggestion
title: BroadcastChannel：messageerror 事件
slug: Web/API/BroadcastChannel/messageerror_event
l10n:
  sourceCommit: 50a45d52fd9f45f1ca30b546af5920d0ccda82dc
```

File: files/zh-cn/web/api/broadcastchannel/postmessage/index.md [hunk 5]
```diff
@@ -1,20 +1,34 @@
 ---
-title: BroadcastChannel.postMessage()
+title: BroadcastChannel：postMessage() 方法
 slug: Web/API/BroadcastChannel/postMessage
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcastchannel/postmessage/index.md:3]:
```suggestion
title: BroadcastChannel：postMessage() 方法
slug: Web/API/BroadcastChannel/postMessage
l10n:
  sourceCommit: 50a45d52fd9f45f1ca30b546af5920d0ccda82dc
```

File: files/zh-cn/web/api/broadcastchannel/close/index.md [hunk 6]
```diff
@@ -1,27 +1,27 @@
...
-```plain
-var str = channel.close();
+```js-nolint
+close()
 ```
 
 ## 示例
 
 ```js
 // 连接到指定频道
```
Comment (jasonren0403) [inline files/zh-cn/web/api/broadcastchannel/close/index.md:21]: 频道还是通道（L10），要统一下

File: files/zh-cn/web/api/broadcastchannel/messageerror_event/index.md [hunk 7]
```diff
@@ -50,7 +55,7 @@ channel.addEventListener("messageerror", (event) => {
 });
 ```
 
-使用 [`onmessage`](/zh-CN/docs/Web/API/BroadcastChannel/onmessage) 和 [`onmessageerror`](/zh-CN/docs/Web/API/BroadcastChannel/onmessageerror) 事件处理程序来实现相同效果：
+相同，但使用 `onmessage` 和 `onmessageerror` 事件处理器属性：
```
Comment (jasonren0403) [inline files/zh-cn/web/api/broadcastchannel/messageerror_event/index.md:58]:
```suggestion
下述代码也实现了相同需求，但使用 `onmessage` 和 `onmessageerror` 事件处理器属性：
```

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
需要留意下该接口下 channel 一词的翻译

File: files/zh-cn/web/api/broadcastchannel/close/index.md [hunk 8]
```diff
@@ -1,27 +1,27 @@
...
+  sourceCommit: 50a45d52fd9f45f1ca30b546af5920d0ccda82dc
 ---
 
-{{APIRef("BroadCastChannel API")}}
+{{APIRef("BroadCastChannel API")}} {{AvailableInWorkers}}
 
-通过调用 **`BroadcastChannel.close()`** 方法，可以马上断开其与对应频道的关联，并让其被垃圾回收。这是必要的步骤，因为浏览器没有其他方式知道频道不再被需要。
-
-{{AvailableInWorkers}}
+{{domxref("BroadcastChannel")}} 接口的 **`close()`** 方法终止与底层通道的连接，从而允许对对象进行垃圾回收。这是必须执行的步骤，因为浏览器没有其他方法可以知道不再需要此通道。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcastchannel/close/index.md:10]:
```suggestion
{{domxref("BroadcastChannel")}} 接口的 **`close()`** 方法终止与底层频道的连接，从而允许对对象进行垃圾回收。这是必须执行的步骤，因为浏览器没有其他方法可以知道不再需要此频道。
```

File: files/zh-cn/web/api/broadcastchannel/messageerror_event/index.md [hunk 9]
```diff
@@ -1,42 +1,47 @@
...
-            >onmessageerror</a
-          ></code
-        >
-      </td>
-    </tr>
-  </tbody>
-</table>
+{{APIRef("BroadCastChannel API")}}{{AvailableInWorkers}}
+
+{{domxref("BroadcastChannel")}} 接口的 **`messageerror`** 事件在无法反序列化的消息到达通道时触发。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/broadcastchannel/messageerror_event/index.md:10]:
```suggestion
{{domxref("BroadcastChannel")}} 接口的 **`messageerror`** 事件在无法反序列化的消息到达频道时触发。
```

**Review by jasonren0403 — APPROVED** [review]

**Pull request merged**